bidon_api/config/environments/production.rb

```diff
@@ -78,5 +78,5 @@
   # Do not dump schema after migrations.
   # config.active_record.dump_schema_after_migration = false
 
-  config.hosts << 'b.appbaqend.com'
+  config.hosts << ENV.fetch('HOST')
```

Comment:
What if we have multiple hosts served by single server? Like we have in appodeal